services/blockchain-app-registry/shared/utils/downloadUtils.js

```diff
@@ -46,15 +51,20 @@ const downloadAndExtractTarball = (url, directoryPath) => new Promise((resolve,
 	});
 });
 
-const downloadFile = (url, dirPath) => new Promise((resolve, reject) => {
+const downloadFile = (url, filePath) => new Promise((resolve, reject) => {
+	if (!url || !filePath) {
+		reject(new Error(`Invalid url or filePath. url: ${url} filePath:${filePath}`));
+		return;
```

Comment:
```suggestion
		return reject(new Error(`Invalid url or filePath. url: ${url} filePath:${filePath}`));
```

Reply:
Actually the function is void, the return won't be used. Also, the linter complains because of inconsistent return statement